lib/index.js

```diff
@@ -269,14 +271,14 @@ internals.knex = (getServer, realmPath) => {
 };
 
 internals.state = (srv) => {
```

Comment:
As written this is no longer a server, but rather a `realm`.  We can chat about it tomorrow.

Reply:
sweet. thanks for catching, my dude 🙏 sounds good